translations/en.json

```diff
@@ -753,6 +802,7 @@
   "nistsp80053": "NIST SP 800-53",
   "nistsp800171": "NIST SP 800-171",
   "no": "No",
+  "noActionRequired": "<em>No Action Required</em>",
```

Comment:
`em` tag can be wrapped at the usage point

Reply:
Done